src/main/java/firebase/sdk/dashboard/api/PlatformResource.java

```diff
@@ -0,0 +1,55 @@
+package firebase.sdk.dashboard.api;
+
+import java.util.List;
+import java.util.HashMap;
+import java.util.Date;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+import javax.ws.rs.core.MediaType;
+import firebase.sdk.dashboard.dao.PlatformReleaseDao;
+import firebase.sdk.dashboard.dao.PlatformReleaseDaoDatastore;
+import firebase.sdk.dashboard.dao.SDKDao;
+
+/**
+ * Root resource (exposed at "myresource" path)
+ */
+@Path("platforms")
+@Produces(MediaType.APPLICATION_JSON)
+public class PlatformResource {
+
+  //private static final PlatformReleaseDao PRDAO = new PlatformReleaseDaoDatastore();
+//  private static final SDKDao SDAO = new SDKDao();
+
+  /**
+   * Method handling HTTP GET requests. The returned object will be sent
```

Comment:
I think the Javadoc will change when you DAO & Data objects. Add a TODO so that you dont forget